app.rb

```diff
@@ -119,6 +126,8 @@ def create_rental
         puts 'Invalid date format. Please use yyyy-mm-dd.'
       end
     end
+
+    start
   end
 
   def list_all_rentals
```

Comment:
- [ ] Well done :clap: you fixed the errors suggested by the previous :point_up:  reviewer :+1: but there is one new error and the error is `undefined method 'id'` so kindly fix this error as well :slightly_smiling_face: 

![Screenshot from 2023-09-29 17-37-03](https://github.com/Eccb7/school-library/assets/45226848/db5020af-de68-4444-ac38-153ab0881a21)